lib/inets/src/http_lib/http_transport.erl

```diff
@@ -220,19 +164,14 @@ accept(ip_comm, ListenSocket, Timeout) ->
 accept({ip_comm, _}, ListenSocket, Timeout) ->
     gen_tcp:accept(ListenSocket, Timeout);
 
-%% Wrapper for backaward compatibillity
-accept({ssl, SSLConfig}, ListenSocket, Timeout) ->
-    accept({?HTTP_DEFAULT_SSL_KIND, SSLConfig}, ListenSocket, Timeout);
-
-accept({essl, _SSLConfig}, ListenSocket, Timeout) ->
+accept({ssl, _SSLConfig}, ListenSocket, Timeout) ->
     ssl:transport_accept(ListenSocket, Timeout).
 
-
 %%-------------------------------------------------------------------------
 %% Description: Assigns a new controlling process to Socket.
 %%-------------------------------------------------------------------------
 -spec controlling_process(SocketType, Socket, NewOwner) -> Object when
-      SocketType :: ip_comm | {ip_comm | ssl | essl, _SSLConfig},
+      SocketType :: ip_comm | {ssl, _SSLConfig},
```

Comment:
the `{ip_comm, _}` type should be kept, as there is a function header that pattern matches on it

Reply:
fixed